transmogrifier/sources/ead.py

```diff
@@ -42,10 +95,22 @@ def get_main_titles(cls, xml: Tag) -> list[str]:
         Args:
             xml: A BeautifulSoup Tag representing a single EAD XML record.
         """
-        return [
-            ", ".join(string for string in titleproper.stripped_strings)
-            for titleproper in xml.find_all("titleproper")
-        ]
+        main_titles = []
+        if collection_description := xml.metadata.find("archdesc", level="collection"):
+            collection_description_did = collection_description.find("did")
+            if collection_description_did:
+                main_titles = [
+                    unittitle_value
+                    for unittitle_elem in collection_description_did.find_all(
+                        "unittitle", recursive=False
+                    )
+                    if (
+                        unittitle_value := ", ".join(
```

Comment:
I believe that value is also represented in the `unitid` for most records so I'll take a look at adding that when I do `identifiers` or whether `unittitle > num` just going to be a dupe value for almost every record

Reply:
Yeah sorry if that wasn't clear...ASpace is definitely adding the unit ID to the title during export. So we don't need to do anything with it here except exclude it from the title, bc as you say it will also be present in the unitid.

Reply:
👍